tests/pytest_tests/system_tests/test_launch/test_launch_kubernetes.py

```diff
@@ -37,6 +37,7 @@ def test_kubernetes_run_clean_generate_name(relay_server, monkeypatch, assets_pa
         project.target_project = project_name
         project.override_config = {}
         project.job = "testjob"
+        project.launch_spec = {"_resume_count": 0}
```

Comment:
nit: I don't think this change should impact this test, if it does, I think it points to a bug

Reply:
`get_env_vars_dict` gets `_resume_count` from the launch spec (defaulting to 0) and checks if it's >0. But if I don't explicitly define launch_spec here, it sees that `_resume_count` is a MagicMock and throws an error when it tries to compare to 0.

Although, yeah, it could have just been `project.launch_spec = {}`.